.eslintrc

```diff
@@ -9,10 +13,24 @@
   ],
   "plugins": [
     "flowtype",
-    "prettier"
+    "prettier",
+    "jest"
   ],
   "rules": {
-    "prettier/prettier": "error",
+    "prettier/prettier": [
+      "error",
+      {
+        "bracketSpacing": true,
+        "printWidth": 80,
+        "singleQuote": false,
+        "tabWidth": 2,
+        "trailingComma": "all"
+      }
+    ],
+    "import/no-extraneous-dependencies": "off",
+    "react/jsx-filename-extension": "off",
```

Comment:
why?

Reply:
I would agree with turning this off, I don't see any reason to make difference whether file has some JSX or not, it's like prefixing all string variables with `s`...

Reply:
Changed to `.js` only. I don't see benefit to use `.jsx`.

Reply:
well JSX is not a valid JS syntax, that's a pretty simple reason. whatever, you guys are the majority